programs/sp1/mock-update-client/Cargo.toml

```
@@ -0,0 +1,13 @@
+cargo-features = ["edition2024"]
+[package]
+name = "mock-update-client"
+version = "0.1.0"
+edition = "2024"
```

this requires nightly Rust but if I downgrade to edition 2021 it still works

```suggestion
[package]
name = "mock-update-client"
version = "0.1.0"
edition = "2021"
```

Rust 2024 hasn't been released yet: https://doc.rust-lang.org/edition-guide/rust-2024/index.html so proposal to use latest stable